src/KOKKOS/verlet_kokkos.cpp

```diff
@@ -140,11 +141,10 @@ void VerletKokkos::setup(int flag)
     atomKK->sync(force->pair->execution_space,force->pair->datamask_read);
     force->pair->compute(eflag,vflag);
     atomKK->modified(force->pair->execution_space,force->pair->datamask_modify);
-    timer->stamp(Timer::PAIR);
   }
   else if (force->pair) force->pair->compute_dummy(eflag,vflag);
 
-  if (atomKK->molecular) {
+  if (atomKK->molecular != Atomic::ATOMIC) {
```

Comment:
@stanmoore1 Probably should be:
```suggestion
if (atomKK->molecular != Atom::ATOMIC) {
```

Reply:
@rbberger True, thanks!